jerry-core/debugger/debugger.c

```diff
@@ -522,6 +523,22 @@ jerry_debugger_breakpoint_hit (uint8_t message_type) /**< message type */
 } /* jerry_debugger_breakpoint_hit */
 
 /**
+* Suspend execution for received time
+*/
+void jerry_debugger_sleep (int milliseconds) /**< Suspending time */
+{
+  #ifdef HAVE_TIME_H
+  nanosleep (&(const struct timespec)
+  {
+    milliseconds / 1000, milliseconds % 1000
+  }
+  , NULL);
+  #elif defined (HAVE_UNISTD_H)
+  usleep (milliseconds * 1000);
+  #endif
```

Comment:
`#endif` (and `#else`) directives are (almost) always followed by comments (here, it should be `/* HAVE_TIME */`, if I'm not mistaken. please, grep around in jerry-core for `#endif`)